src/components/Asset.vue

```diff
@@ -63,6 +70,8 @@ export default {
     },
   },
   computed: {
+    altTextId: ({ _uid }) => `alt-video-text-${_uid}`,
```

Comment:
Can we just use our existing `identifier` prop instead of relying on Vue's internal `_uid` value?

Reply:
Good idea! I used the identifier as unique id :)